packages/toolpad-app/src/toolpad/propertyControls/Markdown.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Paper,
+  PaperProps,
+  Skeleton,
+} from '@mui/material';
+import * as React from 'react';
+import Draggable from 'react-draggable';
+import type { EditorProps } from '../../types';
+import useShortcut from '../../utils/useShortcut';
+import lazyComponent from '../../utils/lazyComponent';
+
+function DraggablePaperComponent(props: PaperProps) {
```

Comment:
I had missed this change last time I reviewed.
What does this add, is it to make the markdown modal draggable?
If so, and if that's a feature worth adding a new dependency for, maybe it should be a more global behavior that can be easily added to other similar dialogs?

Reply:
Agreed, keeping this out of here for now. See: https://github.com/mui/mui-toolpad/pull/1298#discussion_r1030314930